payments-ui-core/res/values/totranslate.xml

```diff
@@ -5,4 +5,7 @@
     -->
     <!-- Card details section title for card form entry -->
     <string name="card_information" tools:ignore="ExtraTranslation">Card information</string>
+
+    <!-- Scan card button -->
+    <string name="scan_card" tools:ignore="ExtraTranslation">Scan card</string>
```

Comment:
I am looking up the translated string in localize and adding it.

Reply:
ok, got it in localize, and it will update the next time we run the script.  Best to wait until this other PR is merged to run the localize script again. https://github.com/stripe/stripe-android/pull/4892